Library/OcConsoleLib/TextOutputBuiltin.c

```diff
@@ -620,6 +82,89 @@ STATIC EFI_CONSOLE_CONTROL_SCREEN_MODE      mConsoleMode = EfiConsoleControlScre
 #define MIN_SUPPORTED_CONSOLE_WIDTH   (80)
 #define MIN_SUPPORTED_CONSOLE_HEIGHT  (25)
 
+BOOLEAN
```

Comment:
Another note to self: This may need EFI_STATUS, and a different status for 'returned usable fallback character' vs. 'could not return anything'?